### PR TITLE
chore: add deprecated warning message

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 📝 Changelog

chore: add deprecated warning message

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | chore: add deprecated warning message |
| 🇨🇳 Chinese | 增加error信息 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
加个用例

**Comment by li-jia-nan** [comment]
> 加个用例

done

File: components/back-top/__tests__/index.test.tsx [hunk 1]
```diff
@@ -43,4 +43,10 @@ describe('BackTop', () => {
     fireEvent.click(container.querySelector('.ant-back-top')!);
     expect(onClick).toHaveBeenCalled();
   });
+
+  it('should throw Error', () => {
+    const errSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+    render(<BackTop />);
+    expect(errSpy).toHaveBeenCalled();
```
Comment (MadCcc) [inline components/back-top/__tests__/index.test.tsx:50]:
```suggestion
    expect(errSpy).toHaveBeenCalled();
    errSpy.mockRestore();
```

Comment (li-jia-nan) [reply components/back-top/__tests__/index.test.tsx:50]: done

**Comment by MadCcc** [comment]
BackTop 的文档也可以移除了

File: components/back-top/__tests__/index.test.tsx [hunk 2]
```diff
@@ -43,4 +43,11 @@ describe('BackTop', () => {
...
```
Comment (MadCcc) [inline components/back-top/__tests__/index.test.tsx:50]: 用 `HaveBeenCalledWith` 检查好些

Comment (li-jia-nan) [reply components/back-top/__tests__/index.test.tsx]: done

**Comment by li-jia-nan** [comment]
> BackTop 的文档也可以移除了

done

**Comment by afc163** [comment]
原文档链接是不是直接跳到 float-button

**Pull request closed without merging**